Firestore/Example/SwiftBuildTest/main.swift

```diff
@@ -198,13 +198,13 @@ func readDocument(at docRef: DocumentReference) {
       if let data = document.data() {
         print("Read document: \(data)")
       }
-      if let data = document.data(with: SnapshotOptions.serverTimestampBehavior(.estimate)) {
+      if let data = document.data(with: .estimate) {
```

Comment:
Should we implement what's in the API proposal until we officially decide to do something different? (at which point we may need to do a cherry-pick or whatever)

I don't care too much as long as we're careful not to propose one thing and then submit something else...

Reply:
Good idea. I'll do that.